packages/framework-integration-tests/integration/provider-specific/local/stop/setup.ts

```diff
@@ -0,0 +1,15 @@
+import { removeFolders } from '../../../helper/file-helper'
+import { sandboxName } from '../constants'
+import { sandboxPathFor } from '../../../helper/file-helper'
+import { readFileSync } from 'fs'
+import * as path from 'path'
+
+before(async () => {
+  const sandboxPath = sandboxPathFor(sandboxName)
+  const pidFile: string = path.join(sandboxPathFor(sandboxName), 'local_provider.pid')
+  const pid = parseInt(readFileSync(pidFile).toString(), 10)
+  console.log(`stopping local server with pid ${pid}...`)  
+  process.kill(pid)
```

Comment:
Killing the process with the PID we stored in `start` is enough to stop the Express server of the Local Provider